src/ert/shared/services/storage_service.py

```diff
@@ -41,6 +43,16 @@ def fetch_auth(self) -> Tuple[str, Any]:
         """
         return ("__token__", self.fetch_conn_info()["authtoken"])
 
+    # @classmethod
+    # def start_service(cls, *args: Any, **kwargs: Any):
+    #     try:
+    #         service = cls.connect(timeout=0, project=kwargs.get("project"))
+    #         # Attempt to fetch the service url
+    #         _ = service.fetch_url()
+    #     except TimeoutError:
+    #         return cls.start_server(*args, **kwargs)
+    #     return _Context(service)
+
```

Comment:
were these commented code lines left by accident?

Reply:
Yes, I forgot to stash the changes before committing  :+1: